classes/03class/exercises/c03-docker02/mrcsmonteiro/README.md

```diff
@@ -0,0 +1,42 @@
+# C03-Docker02
+
+## Docker 
+- [Dockerfile.v1](Dockerfile.v1)
+- [Dockerfile.v2](Dockerfile.v2)
+
+## Command Execution Output
+- Curl command and its output of before changing the image:
+```
+# Build & run commands
+
+$ docker build -t exercise02:v1 . -f Dockerfile.v1
+
+$ docker run -dit -p 8081:80 exercise02:v1
```

Comment:
what is the `-dit` doing?

Reply:
Unintended copy/paste. 'it' removed from the parameter.